commands/issue.go

```diff
@@ -710,3 +720,105 @@ func milestoneValueToNumber(value string, client *github.Client, project *github
 
 	return 0, fmt.Errorf("error: no milestone found with name '%s'", value)
 }
+
+func transferIssue(cmd *Command, args *Args) {
+	if args.ParamsSize() < 2 {
+		ui.Errorln("Usage: hub issue transfer <ISSUE_NUMBER> <TARGET_REPO>")
+		os.Exit(1)
+	}
+
+	issueNumber := args.GetParam(0)
+	targetRepo := args.GetParam(1)
+
+	var issueID, repositoryID string
+
+	localRepo, err := github.LocalRepo()
+	utils.Check(err)
+
+	project, err := localRepo.MainProject()
+	utils.Check(err)
+
+	owner := project.Owner
+	currRepo := project.Name
+	host := project.Host
+	gh := github.NewClient(host)
+
+	query := fmt.Sprintf(`
+        query($issue: Int = %s, $owner: String = "%s", $repo: String = "%s") {
+            repository(owner: $owner, name: $repo) {
+                issue(number: $issue) {
+                    id
+                }
+            }
+        }`, issueNumber, owner, currRepo)
+	body := make(map[string]interface{})
+	body["query"] = query
+
+	response, err := gh.GenericAPIRequest("POST", "graphql", body, nil, 0)
+	utils.Check(err)
+
+	responseData := make(map[string]interface{})
```

Comment:
You can have this be:

```go
responseData := struct {
  Data struct {
    Repository struct {
      Issue struct {
        ID string `json:"id"`
      }
    }
  }
  Errors []struct {
    Message string
  }
}{}
```

And then you can umarshal into that and read `responseData.Data.Repository.Issue.ID` directly without using type assertions.